docs/BuildOptions.md

```diff
@@ -174,275 +153,236 @@ as the `inet_pton` function gets a poor-man's simple replacement that can
 only resolve numeric IPv4 addresses.
 
 
-`--enable-logging` (default: ON)
--------------------------------
+**`--enable-logging`** (default: ON)
 
-Enables logging. When you turn this option OFF, logging will not be
-done at all. Could be tried when you suspect the logging system of
-impairing the performance, at the expense of not having some information
-about errors happening during runtime.
+Enables logging. When you turn this option OFF, the library will not report
+any runtime information through the logging system, including errors.
+This option may be useful if you suspect the logging system of
+impairing performance.
 
 
-`--enable-monotonic-clock` (default: OFF)
------------------------------------------
-
-Enforced `clock_gettime` with monotonic clock on Garbage Collector CV.
-**This is a temporary fix for #729**: The library could got stuck when you
-modify the system clock while a transmission is done over SRT.
+**`--enable-monotonic-clock`** (default: OFF)
 
 This option enforces the use of `clock_gettime` to get the current
-time, instead of `gettimeofday`. This function allows to force a monotonic
-clock, which is independent on the currently set time in the system. The CV,
-for which the `*_timedwait()` functions are used with so obtained current time,
-must be appropriately configured (and this is done so for now only for the
-GarbageCollector controlling CV, not every CV used in SRT). The consequence of
-enabling this option, however, may be portability issues around the
-`clock_gettime()` function, which is not available on every SDK, or extra
+time, instead of `gettimeofday`. This function forces the use of a monotonic
+clock that is independent of the currently set time in the system.
+The condition variables, for which the `*_timedwait()` functions are used
+with time specification basing on the time obtained from `clock_gettime`
+must be appropriately configured. For now, this is only done for the
+GarbageCollector controlling CV, not every CV used in SRT. The consequence
+of enabling this option, however, may be portability issues resulting from
+the fact that `clock_gettime` function may be unavailable in some SDKs or an extra
```

Comment:
the fact that `clock_gettime` function may be unavailable in some SDKs, or that an extra